py/tests/test_case.py

```diff
@@ -0,0 +1,11 @@
+import unittest
+import os
+import sys
+
+from tests.loading_utils import * # pylint: disable=C0413
```

Comment:
what error are you disabling here? When I take the comment out I don't get an error